```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/j6k7sm7mgb9hcz9zpmbhg9mghyahgz1z-php-8.0.13.tar.bz2
source root is php-8.0.13/ext/xmlreader
setting SOURCE_DATE_EPOCH to timestamp 1637086041 of file php-8.0.13/ext/xmlreader/tests/static.phpt
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
/build/php-8.0.13 /build/php-8.0.13/ext/xmlreader
/build/php-8.0.13/ext/xmlreader
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
Configuring for:
PHP Api Version:         20200930
Zend Module Api No:      20200930
Zend Extension Api No:   420200930
configure.ac:18: warning: $as_echo is obsolete; use AS_ECHO(["message"]) instead
build/php.m4:2072: PHP_CONFIG_NICE is expanded from...
configure.ac:18: the top level
configure.ac:161: warning: The macro `AC_LANG_C' is obsolete.
configure.ac:161: You should run autoupdate.
./lib/autoconf/c.m4:72: AC_LANG_C is expanded from...
build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: The macro `AC_LANG_C' is obsolete.
configure.ac:161: You should run autoupdate.
./lib/autoconf/c.m4:72: AC_LANG_C is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2249: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2270: AC_CACHE_CHECK is expanded from...
build/libtool.m4:561: _LT_AC_LOCK is expanded from...
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: The macro `AC_TRY_LINK' is obsolete.
configure.ac:161: You should run autoupdate.
./lib/autoconf/general.m4:2920: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:692: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
./lib/autoconf/general.m4:2249: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2270: AC_CACHE_CHECK is expanded from...
build/libtool.m4:561: _LT_AC_LOCK is expanded from...
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me:${as_lineno-$LINENO}: WARNING: \`$CC' does not support \`-c -o', so \`make -j' may be unsafe
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me: WARNING: \`$CC' does not support \`-c -o', so \`make -j' may be unsafe
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me:${as_lineno-$LINENO}: WARNING: output file \`$ofile' does not exist
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me: WARNING: output file \`$ofile' does not exist
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me:${as_lineno-$LINENO}: WARNING: output file \`$ofile' does not look like a libtool script
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me: WARNING: output file \`$ofile' does not look like a libtool script
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me:${as_lineno-$LINENO}: WARNING: using \`LTCC=$LTCC', extracted from \`$ofile'
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me: WARNING: using \`LTCC=$LTCC', extracted from \`$ofile'
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: tag name \"$tagname\" already exists
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: The macro `AC_LANG_CPLUSPLUS' is obsolete.
configure.ac:161: You should run autoupdate.
./lib/autoconf/c.m4:262: AC_LANG_CPLUSPLUS is expanded from...
build/libtool.m4:2810: _LT_AC_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:2809: AC_LIBTOOL_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me:${as_lineno-$LINENO}: WARNING: \`$CC' does not support \`-c -o', so \`make -j' may be unsafe
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2810: _LT_AC_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:2809: AC_LIBTOOL_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
configure.ac:161: warning: back quotes and double quotes must not be escaped in: $as_me: WARNING: \`$CC' does not support \`-c -o', so \`make -j' may be unsafe
build/libtool.m4:1185: AC_LIBTOOL_SYS_HARD_LINK_LOCKS is expanded from...
build/libtool.m4:2810: _LT_AC_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:2809: AC_LIBTOOL_LANG_CXX_CONFIG is expanded from...
build/libtool.m4:1918: _LT_AC_TAGCONFIG is expanded from...
build/libtool.m4:70: AC_LIBTOOL_SETUP is expanded from...
build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
fixing libtool script ./build/ltmain.sh
configure flags: --disable-static --prefix=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13 --bindir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/bin --sbindir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/sbin --includedir=/nix/store/228cv4xpgawwf5dxdrpr9ag8skq7y2a0-php-xmlreader-8.0.13-dev/include --oldincludedir=/nix/store/228cv4xpgawwf5dxdrpr9ag8skq7y2a0-php-xmlreader-8.0.13-dev/include --mandir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/share/man --infodir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/share/info --docdir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/share/doc/php-xmlreader --libdir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/lib --libexecdir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/libexec --localedir=/nix/store/10q8smfyq5s1wssaskrqsxr570yzfl1z-php-xmlreader-8.0.13/share/locale --enable-xmlreader
checking for grep that handles long lines and -e... /nix/store/iz7g111rvlqwjsr8qlpnb3p5y44ji3kp-gnugrep-3.7/bin/grep
checking for egrep... /nix/store/iz7g111rvlqwjsr8qlpnb3p5y44ji3kp-gnugrep-3.7/bin/grep -E
checking for a sed that does not truncate output... /nix/store/220vwl17jy619jfrmifarw09sgfdkvr9-gnused-4.8/bin/sed
checking pkg-config is at least version 0.9.0... yes
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking how to run the C preprocessor... gcc -E
checking for icc... no
checking for suncc... no
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for PHP prefix... /nix/store/abv5xnf1mm9z93y7sj3mf14ysgklz2gk-php-8.0.13
checking for PHP includes... -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/main -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/TSRM -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/Zend -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext/date/lib
checking for PHP extension directory... /nix/store/abv5xnf1mm9z93y7sj3mf14ysgklz2gk-php-8.0.13/lib/php/extensions
checking for PHP installed headers prefix... /nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php
checking if debug is enabled... no
checking if zts is enabled... yes
checking for gawk... gawk
checking whether to enable XMLReader support... yes, shared
checking for libxml-2.0 >= 2.9.0... yes
checking for a sed that does not truncate output... /nix/store/220vwl17jy619jfrmifarw09sgfdkvr9-gnused-4.8/bin/sed
checking for ld used by gcc... ld
checking if the linker (ld) is GNU ld... yes
checking for ld option to reload object files... -r
checking for BSD-compatible nm... nm
checking whether ln -s works... yes
checking how to recognize dependent libraries... (cached) pass_all
./configure: line 5638: /usr/bin/file: No such file or directory
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 1572864
checking command to parse nm output from gcc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC
checking if gcc PIC flag -fPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking whether the gcc linker (ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
checking whether the g++ linker (ld) supports shared libraries... yes
checking for g++ option to produce PIC... 
checking if g++ static flag  works... yes
checking if g++ supports -c -o file.o... yes
checking whether the g++ linker (ld) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
(cached) (cached) checking how to hardcode library paths into programs... unsupported
configure: patching config.h.in
configure: creating ./config.status
config.status: creating config.h
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j4 -l4 SHELL=/nix/store/svxmj4xahyjqq4lciisvgi1fwl5bxpdd-bash-5.1-p8/bin/bash
/nix/store/svxmj4xahyjqq4lciisvgi1fwl5bxpdd-bash-5.1-p8/bin/bash /build/php-8.0.13/ext/xmlreader/libtool --mode=compile gcc -I. -I/build/php-8.0.13/ext/xmlreader -I/build/php-8.0.13/ext/xmlreader/include -I/build/php-8.0.13/ext/xmlreader/main -I/build/php-8.0.13/ext/xmlreader -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/main -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/TSRM -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/Zend -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext/date/lib -I/nix/store/mzg93a32bghig2w8c4hjbmqksab8cs3g-libxml2-2.9.12-dev/include/libxml2  -DHAVE_CONFIG_H  -g -O2    -c /build/php-8.0.13/ext/xmlreader/php_xmlreader.c -o php_xmlreader.lo 
mkdir .libs
 gcc -I. -I/build/php-8.0.13/ext/xmlreader -I/build/php-8.0.13/ext/xmlreader/include -I/build/php-8.0.13/ext/xmlreader/main -I/build/php-8.0.13/ext/xmlreader -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/main -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/TSRM -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/Zend -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext/date/lib -I/nix/store/mzg93a32bghig2w8c4hjbmqksab8cs3g-libxml2-2.9.12-dev/include/libxml2 -DHAVE_CONFIG_H -g -O2 -c /build/php-8.0.13/ext/xmlreader/php_xmlreader.c  -fPIC -DPIC -o .libs/php_xmlreader.o
/nix/store/svxmj4xahyjqq4lciisvgi1fwl5bxpdd-bash-5.1-p8/bin/bash /build/php-8.0.13/ext/xmlreader/libtool --mode=link gcc -shared -I/build/php-8.0.13/ext/xmlreader/include -I/build/php-8.0.13/ext/xmlreader/main -I/build/php-8.0.13/ext/xmlreader -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/main -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/TSRM -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/Zend -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext -I/nix/store/8d84ys2j4dd9wmwah3gqncvi3xjfkhb7-php-8.0.13-dev/include/php/ext/date/lib -I/nix/store/mzg93a32bghig2w8c4hjbmqksab8cs3g-libxml2-2.9.12-dev/include/libxml2  -DHAVE_CONFIG_H  -g -O2    -o xmlreader.la -export-dynamic -avoid-version -prefer-pic -module -rpath /build/php-8.0.13/ext/xmlreader/modules  php_xmlreader.lo -Wl,-rpath,/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib -L/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib -lxml2
gcc -shared  .libs/php_xmlreader.o  -Wl,--rpath -Wl,/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib -Wl,--rpath -Wl,/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib -L/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib /nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib/libxml2.so  -Wl,-rpath -Wl,/nix/store/s98s7bzlgsq5m1xqylv6nazsp8mzf330-libxml2-2.9.12/lib -Wl,-soname -Wl,xmlreader.so -o .libs/xmlreader.so
creating xmlreader.la
(cd .libs && rm -f xmlreader.la && ln -s ../xmlreader.la xmlreader.la)
/nix/store/svxmj4xahyjqq4lciisvgi1fwl5bxpdd-bash-5.1-p8/bin/bash /build/php-8.0.13/ext/xmlreader/libtool --mode=install cp ./xmlreader.la /build/php-8.0.13/ext/xmlreader/modules
cp ./.libs/xmlreader.so /build/php-8.0.13/ext/xmlreader/modules/xmlreader.so
cp ./.libs/xmlreader.lai /build/php-8.0.13/ext/xmlreader/modules/xmlreader.la
PATH="$PATH:/sbin" ldconfig -n /build/php-8.0.13/ext/xmlreader/modules
----------------------------------------------------------------------
Libraries have been installed in:
   /build/php-8.0.13/ext/xmlreader/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the `-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the `LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the `LD_RUN_PATH' environment variable
     during linking
   - use the `-Wl,--rpath -Wl,LIBDIR' linker flag

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------

Build complete.
Don't forget to run 'make test'.

@nix { "action": "setPhase", "phase": "checkPhase" }
running tests

Build complete.
Don't forget to run 'make test'.


=====================================================================
PHP         : /nix/store/abv5xnf1mm9z93y7sj3mf14ysgklz2gk-php-8.0.13/bin/php 
Warning: PHP Startup: Unable to load dynamic library 'xmlreader.so' (tried: /build/php-8.0.13/ext/xmlreader/modules/xmlreader.so (/build/php-8.0.13/ext/xmlreader/modules/xmlreader.so: undefined symbol: dom_node_class_entry), /build/php-8.0.13/ext/xmlreader/modules/xmlreader.so.so (/build/php-8.0.13/ext/xmlreader/modules/xmlreader.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

PHP_SAPI    : cli
PHP_VERSION : 8.0.13
ZEND_VERSION: 4.0.13
PHP_OS      : Linux - Linux localhost 5.10.69 #1-NixOS SMP Sun Sep 26 12:09:02 UTC 2021 x86_64
INI actual  : /build/php-8.0.13/ext/xmlreader/tmp-php.ini
More .INIs  :   
---------------------------------------------------------------------
PHP         : /nix/store/abv5xnf1mm9z93y7sj3mf14ysgklz2gk-php-8.0.13/bin/phpdbg 
PHP_SAPI    : 
Warning: PHP Startup: Unable to load dynamic library 'xmlreader.so' (tried: /build/php-8.0.13/ext/xmlreader/modules/xmlreader.so (/build/php-8.0.13/ext/xmlreader/modules/xmlreader.so: undefined symbol: dom_node_class_entry), /build/php-8.0.13/ext/xmlreader/modules/xmlreader.so.so (/build/php-8.0.13/ext/xmlreader/modules/xmlreader.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
phpdbg
PHP_VERSION : 8.0.13
ZEND_VERSION: 4.0.13
PHP_OS      : Linux - Linux localhost 5.10.69 #1-NixOS SMP Sun Sep 26 12:09:02 UTC 2021 x86_64
INI actual  : /build/php-8.0.13/ext/xmlreader/tmp-php.ini
More .INIs  : 
---------------------------------------------------------------------
CWD         : /build/php-8.0.13/ext/xmlreader
Extra dirs  : 
VALGRIND    : Not used
=====================================================================
TIME START 2021-11-19 20:05:45
=====================================================================
TEST 1/38 [tests/001.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, string data [tests/001.phpt] 
TEST 2/38 [tests/002.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, file data [tests/002.phpt] 
TEST 3/38 [tests/003-get-errors.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, Get an attribute, with invalid arguments [tests/003-get-errors.phpt] 
TEST 4/38 [tests/003-mb.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, attributes test [tests/003-mb.phpt] 
TEST 5/38 [tests/003-move-errors.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, Move cursor to an attribute, with invalid arguments [tests/003-move-errors.phpt] 
TEST 6/38 [tests/003.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, attributes test [tests/003.phpt] 
TEST 7/38 [tests/004.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, attributes test [tests/004.phpt] 
TEST 8/38 [tests/005.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, parser property set/get [tests/005.phpt] 
TEST 9/38 [tests/006.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, moveToElement [tests/006.phpt] 
TEST 10/38 [tests/007.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, setRelaxNGSchema [tests/007.phpt] 
TEST 11/38 [tests/008.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, DTD [tests/008.phpt] 
TEST 12/38 [tests/009.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, next [tests/009.phpt] 
TEST 13/38 [tests/010.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, next [tests/010.phpt] 
TEST 14/38 [tests/011.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, string data [tests/011.phpt] reason: Fatal error: Uncaught Error: Class "XMLReader" not found in /build/php-8.0.13/ext/xmlreader/tests/011.skip.php:2
TEST 15/38 [tests/012.phpt]
[1;33mSKIP[0m XMLReader: accessing empty and non existing attributes [tests/012.phpt] 
TEST 16/38 [tests/013.phpt]
[1;33mSKIP[0m XMLReader: Schema validation [tests/013.phpt] 
TEST 17/38 [tests/014.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, read-only element values can not be modified [tests/014.phpt] 
TEST 18/38 [tests/015-get-errors.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, Move cursor to a named attribute within a namespace, with invalid arguments [tests/015-get-errors.phpt] 
TEST 19/38 [tests/015-move-errors.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, Move cursor to a named attribute within a namespace, with invalid arguments [tests/015-move-errors.phpt] 
TEST 20/38 [tests/015.phpt]
[1;33mSKIP[0m XMLReader: libxml2 XML Reader, Move cursor to a named attribute within a namespace [tests/015.phpt] 
TEST 21/38 [tests/bug36743.phpt]
[1;33mSKIP[0m Bug #36743 (In a class extending XMLReader array properties are not writable) [tests/bug36743.phpt] 
TEST 22/38 [tests/bug42139.phpt]
[1;33mSKIP[0m Bug #42139 (XMLReader option constants are broken using XML()) [tests/bug42139.phpt] 
TEST 23/38 [tests/bug51936.phpt]
[1;33mSKIP[0m Bug #51936 (Crash with clone XMLReader) [tests/bug51936.phpt] reason: requires xmlreader
TEST 24/38 [tests/bug64230.phpt]
[1;33mSKIP[0m Bug #64230 (XMLReader does not suppress errors) [tests/bug64230.phpt] reason: requires xmlreader
TEST 25/38 [tests/bug70309.phpt]
[1;33mSKIP[0m XMLReader: Bug #70309 XmlReader read generates extra output [tests/bug70309.phpt] reason: .. only for Windows
TEST 26/38 [tests/bug71805.phpt]
[1;33mSKIP[0m XMLReader: Bug #71805 XML files can generate UTF-8 error even if they are UTF-8 [tests/bug71805.phpt] 
TEST 27/38 [tests/bug73053.phpt]
[1;33mSKIP[0m Bug #73053 (XML reader with setSchema now fails under 5.6.25) [tests/bug73053.phpt] reason: xmlreader extension not available
TEST 28/38 [tests/bug73246.phpt]
[1;33mSKIP[0m Bug #73246 (XMLReader: encoding length not checked) [tests/bug73246.phpt] reason: xmlreader extension not available
TEST 29/38 [tests/bug74457.phpt]
[1;33mSKIP[0m XMLReader: Bug #74457 Wrong reflection on XMLReader::expand [tests/bug74457.phpt] reason: xmlreader extension not available
TEST 30/38 [tests/bug81521.phpt]
[1;31mFAIL[0m Bug #81521: XMLReader::getParserProperty may throw with a valid property [tests/bug81521.phpt] 
TEST 31/38 [tests/expand.phpt]
[1;33mSKIP[0m XMLReader: Expand into existing DOM documet [tests/expand.phpt] reason: skip DOM extension required
TEST 32/38 [tests/expand_error.phpt]
[1;33mSKIP[0m XMLReader: Expand Error [tests/expand_error.phpt] reason: skip DOM extension required
TEST 33/38 [tests/next_basic.phpt]
[1;33mSKIP[0m XMLReader: next basic [tests/next_basic.phpt] reason: XMLReader extension required
TEST 34/38 [tests/open_error.phpt]
[1;33mSKIP[0m XMLReader: open error [tests/open_error.phpt] reason: XMLReader extension required
TEST 35/38 [tests/readString_basic.phpt]
[1;33mSKIP[0m XMLReader: readString basic [tests/readString_basic.phpt] reason: XMLReader extension required
TEST 36/38 [tests/setParserProperty_error.phpt]
[1;33mSKIP[0m XMLReader: setParserProperty Error [tests/setParserProperty_error.phpt] reason: XMLReader extension required
TEST 37/38 [tests/setSchema_error.phpt]
[1;33mSKIP[0m XMLReader: setSchema Error [tests/setSchema_error.phpt] reason: XMLReader extension required
TEST 38/38 [tests/static.phpt]
[1;33mSKIP[0m Calling XMLReader::open() and ::XML() statically [tests/static.phpt] reason: xmlreader extension not available
=====================================================================
TIME END 2021-11-19 20:05:46

=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    1
Exts tested     :   12
---------------------------------------------------------------------

Number of tests :   38                 1
Tests skipped   :   37 ( 97.4%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    1 (  2.6%) (100.0%)
Tests passed    :    0 (  0.0%) (  0.0%)
---------------------------------------------------------------------
Time taken      :    1 seconds
=====================================================================

=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
Bug #81521: XMLReader::getParserProperty may throw with a valid property [tests/bug81521.phpt]
=====================================================================

You may have found a problem in PHP.
This report can be automatically sent to the PHP QA team at
http://qa.php.net/reports and http://news.php.net/php.qa.reports
This gives us a better understanding of PHP's behavior.
If you don't want to send the report immediately you can choose
option "s" to save it.	You can then email it to qa-reports@lists.php.net later.
Do you want to send this report now? [Yns]: 
Please enter your email address.
(Your address will be mangled so that it will not go out on any
mailinglist in plain text): 
Warning: fsockopen(): php_network_getaddresses: getaddrinfo failed: Name or service not known in /build/php-8.0.13/ext/xmlreader/run-tests.php on line 1144

Warning: fsockopen(): Unable to connect to qa.php.net:80 (php_network_getaddresses: getaddrinfo failed: Name or service not known) in /build/php-8.0.13/ext/xmlreader/run-tests.php on line 1144

The test script was unable to automatically send the report to PHP's QA Team
Please send /build/php-8.0.13/ext/xmlreader/php_test_results_20211119_2005.txt to qa-reports@lists.php.net manually, thank you.
make: *** [Makefile:132: test] Error 1
```

--- a/pkgs/development/interpreters/php/7.4.nix
+++ b/pkgs/development/interpreters/php/7.4.nix
@@ -2,8 +2,8 @@
 
 let
   base = callPackage ./generic.nix (_args // {
-    version = "7.4.25";
-    sha256 = "sha256-J5klcMrz4uUyOrezeFPETBUpsdMeqU2Xdu+pHVp4ExM=";
+    version = "7.4.26";
+    sha256 = "0k803j5wf4jv72px0zqz2z2hxyk2w3jr6xyczy568dx4z2l8i2yn";
   });
 
 in

--- a/pkgs/development/interpreters/php/8.0.nix
+++ b/pkgs/development/interpreters/php/8.0.nix
@@ -2,8 +2,8 @@
 
 let
   base = callPackage ./generic.nix (_args // {
-    version = "8.0.12";
-    sha256 = "sha256-tIhtsd8yLcj7Eo2LNK5+lPb8aC7LKf9PWlkdTen+rb8=";
+    version = "8.0.13";
+    sha256 = "0djqh650clz4fy1zifazf0jq383znksydx23f1s48prrlixrshf2";
   });
 
 in

--- a/pkgs/top-level/php-packages.nix
+++ b/pkgs/top-level/php-packages.nix
@@ -565,6 +565,7 @@ lib.makeScope pkgs.newScope (self: with self; {
           buildInputs = [ libxml2 ];
           internalDeps = [ php.extensions.dom ];
           NIX_CFLAGS_COMPILE = [ "-I../.." "-DHAVE_DOM" ];
+          doCheck = false;
           configureFlags = [ "--enable-xmlreader" ]
             # Required to build on darwin.
             ++ lib.optionals (lib.versionOlder php.version "7.4") [ "--with-libxml-dir=${libxml2.dev}" ];

